extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -73,6 +76,10 @@ export class AmpAdApiHandler {
     this.baseInstance_.applyFillContent(this.iframe_);
     this.intersectionObserver_ =
         new IntersectionObserver(this.baseInstance_, this.iframe_, is3p);
+    this.embedStateApi_ = new SubscriptionApi(
+        this.iframe_, 'send-embed-state', is3p,
+        () => this.sendEmbedInfo_(this.baseInstance_.isInViewport()));
+    this.embedStateApi_.init();
```

Comment:
I suppose not at the moment, switched it to private and made the constructor call it.